### PR TITLE
Build debian package in CI ##build

### PR DESCRIPTION
**Pull request by radare**
**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](../DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](../DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](../../radare2book) with the relevant information (if needed)

**Detailed description**

...

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

**Comment by radare** [comment]
@XVilka can you add support for uploading the last builds in github? 

Reference link: https://github.com/radareorg/radare2-release/issues/34

We may want to have a place to hold the builds for

* Windows (zip)
* macOS  (pkg)
* Linux  (deb)

Eventually we  can add binary builds for android/iphone and  such, but we shouhld have those platforms covered by default  on every commit in CI beore 4.3

**Comment by XVilka** [comment]
I will check later this week.

File: sys/debian.sh [hunk 1]
```diff
@@ -31,7 +31,9 @@ for a in ${PKGDIR}/usr/lib/libr*.so.* ; do
 done
 
 echo "[debian] building radare2 package..."
+./configure
```
Comment (ret2libc) [inline sys/debian.sh]: isn't one `configure` enough? Also, I think it should be moved before the `make install` just few lines above.

Comment (radare) [reply sys/debian.sh]: yeah, but it doesnt hurts, its cleaner to read imho

Comment (radare) [reply sys/debian.sh]: nvm ive seen the issue :D

**Comment by radare** [comment]
where  are   the  files  uploaded? :?

**Comment by XVilka** [comment]
Stop overloading Travis and using hacky external scripts - see the cleaner solution with GitHub actions: https://github.com/radareorg/radare2/pull/16001

**Pull request merged**